src/matchers/matcher/currentDate.matcher.js

```diff
@@ -0,0 +1,25 @@
+import Sugar from 'sugar-date';
+
+class CurrentDateMatcher {
+  isSatisfiedBy(prefix, name) {
+    return prefix === 'f' && name === 'currentDate';
+  }
+
+  match(element) {
+    let currentDate = Sugar.Date.create('now');
```

Comment:
1. It's simple, so maybe you should use it like this:
`const currentDate = Sugar.Date.format(Sugar.Date.create('now'), '{yyyy}-{MM}-{dd}');`

2. Currently, you're supporting only `{yyyy}-{MM}-{dd}`, I think it should be working in the same way as date comparator. So a user can provide dates `dd/mm/yyyy` and so on.

Reply:
1. Yeah good point
2. It supports all data formats, Sugar library can handle it, it's just changed to this format to compare by dates. https://sugarjs.com/dates/